eth/handler.go

```diff
@@ -608,6 +609,10 @@ func (h *handler) BroadcastTransactions(txs types.Transactions) {
 	)
 	// Broadcast transactions to a batch of peers not knowing about it
 	for _, tx := range txs {
+		// Blob transactions are never broadcast, only ever announced
+		if tx.Type() == types.BlobTxType {
+			continue
+		}
```

Comment:
It seems like returning early here will cause no blob txs to ever be announced? Are the announcements triggered elsewhere?

Reply:
Huh, I could have sworn that there's a second loop that does the announces. Misread the code. Good catch again